defaults/main.yml

```diff
@@ -1,54 +1,186 @@
+# Host name on which the BackupPC server is running
+# We just define this variables for naming coherence, no specific need to define a new one.  
+backuppc_srv_ServerHost: '{{ backuppc_srv_server_name }}'
+
+#### General server configuration
+
+#### What to backup and when to do it
+
+# Default list of system folders to backup from a client
+backuppc_srv_RsyncShareName: 
+- /etc    
+- /var
+- /home
+- /root
+## backuppc_srv_RsyncShareName must be a list of folders (rsync method, presently the only tested)
+## See file templates/etc/backuppc/config.pl.j2, lines 1289 to 1312 
+
+# Default list of files, eventually per share, to include in the backup of a client
+# List of directories or files to include in backups. This can be set to a string, a list of strings,
+# or, in the case of multiple shares, a dict of strings or lists. A dict is used to give a list of directories 
+# or files to backup for each share (the share name is the key). If a hash is used, a special key `"*"` means 
+#it applies to all shares that don't have a specific entry.
+#
+# backuppc_srv_BackupFilesOnly: 
+#
+# See README.md for more explenations and example
+
+# List of directories or files to exclude from the backup: 
+# backuppc_srv_BackupFilesExclude: 
+#
+# Syntax is similar than backuppc_srv_BackupFilesOnly
+
+## Global configuration
+#
+# Not userd with Apache 2, only for direct TCP acces
+backuppc_srv_ServerPort: -1
+backuppc_srv_ServerMesgSecret: ''
+# Maximum automatic simultaneous backups
+backuppc_srv_MaxBackups: 4
+# Maximum user requested simultaneous backups
+backuppc_srv_MaxUserBackups: 4
+# Maximum nithly simultaneous jobs
+backuppc_srv_MaxBackupPCNightlyJobs: 2
+# Days to traverse the entire pool
+backuppc_srv_BackupPCNightlyPeriod: 1
+# Max number of old logs files
+backuppc_srv_MaxOldLogFiles: 14
+# Maximum filling of disk (percentage)
+backuppc_srv_DfMaxUsagePct: 95
+# Sleep tiem parameter
+backuppc_srv_TrashCleanSleepSec: 300
+# Several directories needed
+backuppc_srv_LogDir: /var/lib/backuppc/log
+backuppc_srv_InstallDir: /usr/share/backuppc
+backuppc_srv_CgiDir: /usr/share/backuppc/cgi-bin
+
+#### What to backup and when to do it
+## Following parameters can be overwritten in the per client config.pl
+## See backuppc_client role: https://github.com/UdelaRInterior/ansible-backuppc-client/blob/master/README.md
+
+# Default full period and incemental perior (1 week, 1 day)
+backuppc_srv_FullPeriod: 6.97
+backuppc_srv_IncrPeriod: 0.97
+# See config.pl.j2 line 421 to 495
+# backuppc_srv_FullKeepCnt must be set as a value or the elements of a list of exponential values. Ex: 4, 0, 4, 0, 0, 2  ## 8 month history
+backuppc_srv_FullKeepCnt: 1
+
+# State of the backup
+backuppc_srv_BackupsDisable: 0
+# Other possible values are 1: automatic backup disable, manual backup ok, 2: no backup at all 
+
+# Rsync default commands to backup and restore a host
+backuppc_srv_RsyncClientCmd: '$sshPath -x -q -l {{ backuppc_client_user }} $host sudo $rsyncPath $argList+'
```

Comment:
The `backuppc_client_user` variable does not have a default value. This results in an error on installation when the variables `backuppc_srv_RsyncClientCmd` and `backuppc_srv_RsyncClientRestoreCmd` are invoked in `templates/etc/backuppc/config.pl.j2`.

Reply:
Good point. In my playbooks I don't define this variable for the host I'm testing on, abut the role doesn't fail. Maybe `backuppc_client_user` is defined somwherelse (for client, problably) and for some ansible side-effet it's still there. I won't investigate further, you're right. 
This default definition is a legacy one, nopw we have fully separate variables of server and client role, I think II'll put there `backuppc_srv_unix_user` variable. If the client wants to use another value, in client role she will redefine also `backuppc_**_RsyncClientCmd`.

Reply:
This was solved in further versions, [see in v3.1.1](https://github.com/UdelaRInterior/ansible-backuppc/blob/e9b35873c85e9d5fc6c032722c360aaae4625856/defaults/main.yml#L104). We drop out the variable from the client role and propose to backup the server itself: 
```
# Rsync default commands to backup and restore a host
backuppc_srv_RsyncClientCmd: '$sshPath -x -q -l {{ backuppc_srv_unix_user }} $host sudo $rsyncPath $argList+'
backuppc_srv_RsyncClientRestoreCmd: '$sshPath -q -x -l {{ backuppc_srv_unix_user }} $host sudo $rsyncPath $argList+' 
```
Please, resolve converstion if you agree.